### DISCUSSION TITLE
Editing View for a specific RSS feed tab

### DISCUSSION THREAD

**Discussion by pL-uTo** [FreshRSS/FreshRSS#7078] [Q&A] [answered]
1. FreshRSS Settings TAB -> Reading -> Articles to display -> "show all articles"
2. Add RSS feed -> Download all articles -> Read e.g. 5 articles -> From the 5 articles read, select one article and define whether it should be shown "stop showing in the list of downloaded articles"?

Continued in [commnet](https://github.com/FreshRSS/FreshRSS/discussions/7078#discussioncomment-11523018)

**Comment by Alkarex** [comment]
Hello,
That thread contains a mix of discussions and clarifications, so I am not sure what your specific need or use-case is. But I have already answered https://github.com/FreshRSS/FreshRSS/issues/6149#issuecomment-1977430867 You are welcome to send a pull request for *adding conditions for retrieving full content*.
The specific post you link to (after my answer) is quite far from the way FreshRSS works, and does not take into account the previous post.
Let's close here, if there is nothing new. I will copy my answer in the original thread.

**Reply by ZicPL** [reply]
> Hello, That thread contains a mix of discussions and clarifications, so I am not sure what your specific need or use-case is. But I have already answered [#6149 (comment)](https://github.com/FreshRSS/FreshRSS/issues/6149#issuecomment-1977430867) You are welcome to send a pull request for _adding conditions for retrieving full content_. The specific post you link to (after my answer) is quite far from the way FreshRSS works, and does not take into account the previous post. Let's close here, if there is nothing new. I will copy my answer in the original thread.

The author of this discussion refers to my created thread from the past. I closed my issue because I think this discussion is better for discussing the concept with Owner of FreshRSS.

1. **I have been using this app for a year and I'm very grateful to the Owner for creating FreshRSS.** ❤️ 

2. After using FreshRSS for a long time I noticed that the downloaded articles aren't fully manageable. The RSS feed properties forced the FreshRSS app to "Adapt". @Alkarex  wrote to me that app downloads all items from the RSS feed. The user can only filter them (flag them) and specify the number of unread ones. All this was the reason for the concept that I directed to @Alkarex @frenzie. The concept that I wrote about was about manipulating items from downloaded feeds RSS. I heard that these items are in FreshRSS app Database, so I asked to create a feature that would allow to manipulate items (data in DB):
- **"hiding items"** from the feed after an action performed by user (e.g. stops showing them in their interface). I call this "pseudo-deletion" -> deleting itself is impossible and was explained to me as too much interference in core of FreshRSS app.
- Comparison of downloaded items with those that were hidden by user. I decided that existence of pseudo-deletion (hiding items in FreshRSS interface) wouldn't work well in scenario: if there is update of item source in the feed RSS (e.g previously downloaded item has update in source) that user hid. This feature would verify the change of items content and treat the entry as new (if it was hidden, it is auto-visible again). Many article authors republish the same article with updated content. In this case, checksum would be useful.
- Statistics from item feed RSS vs. things from the DataBase. It was supposed to be a very simple tool (already exists in an incomplete form). It was about displaying the digits compilation:

[ XXX / SSS (AAA)]

, where
XXX - number of available items to download from the RSS feed (currently exists in the feed settings).

SSS - number of downloaded items from the RSS feed, but only those displayed in the FreshRSS app interface.

AAA - number of all downloaded items from the RSS feed to app Database (shown + hidden). This simple view would allow users to better control showing and hiding downloaded items from the RSS feed.
- Preview of downloaded and hidden items from the RSS feed to the FreshRSS app + restore "show item". It may happen that user will want to undo changes - unhide item.

To sum up, the above features are related to each other and don't conflict with the principle of RSS feed, and are also not invasive (core modification) for the FreshRSS app. Manipulation of data (downloaded items from the RSS feed to the FreshRSS Database) could also be called "Advanced FreshRSS Display Filters". I can't estimate how time-consuming it is to implement these features, but I think they're a great solution for people who like to control data in their own instances. I'd appreciate if You (owner app) would consider my request positive, because I think FreshRSS is a great app and I'd be very happy if I could help even in a small way to make it even better.

**Reply by Frenzie** [reply]
I for one support pseudo-deletion anyway. :)

**Reply by Alkarex** [reply]
@ZicPL Regarding "hiding items" and unhiding them, this is already possible by marking articles as read or unread. You can define your user interface to show only the unread articles.

Regarding downloading articles, that only makes sense for the *full content retrieval* mechanism. I plan to add a filter to only download full content for articles matching a criteria. Edit: Done https://github.com/FreshRSS/FreshRSS/commit/33fd07f6f26310d4806077cc87bcdf9b8b940e35#commitcomment-150152171

In the future (we might have an issue somewhere already), I would like to add a context menu to articles to manually perform actions such as download full content.

**Reply by Alkarex** [reply]
Now we have a duplicated topic. It was supposed to be closed here. See https://github.com/FreshRSS/FreshRSS/issues/6149#issuecomment-2531617021

**Reply by ZicPL** [reply]
@Alkarex what is [Content retrieval](https://github.com/FreshRSS/FreshRSS/issues/6149#issuecomment-2531617021)
I don't see it in app.

**Reply by pL-uTo** [reply]
> @ZicPL Regarding "hiding items" and unhiding them, this is already possible by marking articles as read or unread. You can define your user interface to show only the unread articles.
> 

I read @ZicPL comment which clearly shows that he didn't write about unread items. @Alkarex Of course, maybe I've wrong impression that you didn't understand his comment, but in my opinion you didn't refer to his statement 🤔 
I'm also grateful for FreshRSS, but ability to leave **selected** read items or articles visible (I don't know what the difference is) sounds great. A list consisting of unread articles is impractical when a particular article has content that you want to return to in the future. If I could still see: unread + only selected read articles in one list, where I hide the rest of the read articles, that would be the solution to my discussion.
Please look at: I've unread articles (highlighted) set to show on my list and read articles (not highlighted). If I could still see both types of messages, but could decide to hide specific read articles, that would be solution to my discussion. I'd have a list of unread articles and a list of read articles (only this ones I want). I don't understand all the features proposed by @ZicPL , but they seem to make sense.

**Reply by Alkarex** [reply]
> A list consisting of unread articles is impractical when a particular article has content that you want to return to in the future

That is exactly what favourites / starred articles are for. And if more granularity is needed, you can take advantage of user labels to put labels on articles you want to come back later to

**Reply by Alkarex** [reply]
Another advantage of sticking to *read / unread / starred (favourites) / labelled (tagged)* is that those concepts are supported by client apps through the API

**Reply by ZicPL** [reply]
> (..) If I could still see: unread + only selected read articles in one list, where I hide the rest of the read articles, that would be the solution to my discussion. Please look at: I've unread articles (highlighted) set to show on my list and read articles (not highlighted). If I could still see both types of messages, but could decide to hide specific read articles, that would be solution to my discussion. I'd have a list of unread articles and a list of read articles (only this ones I want). I don't understand all the features proposed by @ZicPL , but they seem to make sense.

**This is just a concept.**
I don't know how the view that user sees is related to the DataBase. This is a question for @Alkarex

**Reply by pL-uTo** [reply]
> Now we have a duplicated topic. It was supposed to be closed here. See [#6149 (comment)](https://github.com/FreshRSS/FreshRSS/issues/6149#issuecomment-2531617021)

I host FreshRSS on hosting. I read articles via the website. Can I do this? I don't use readers on my computer.

**Reply by Alkarex** [reply]
Please first make sure you have explored the options offered by the existing system: read / unread / starred (favourites) / labelled (tagged).

**Reply by Alkarex** [reply]
> I host FreshRSS on hosting. I read articles via the website. Can I do this? I don't use readers on my computer.

Do what?

**Reply by Frenzie** [reply]
> Another advantage of sticking to read / unread / starred (favourites) / labelled (tagged) is that those concepts are supported by client apps through the API

Just because all clients are useless because Google Reader is atrocious doesn't mean the web UI has to be imho.

Edit: but I have nothing worth adding over https://github.com/FreshRSS/FreshRSS/issues/1167#issuecomment-480514125

It hasn't crossed into my annoyance threshold any more or less since. ;-)

**Reply by ZicPL** [reply]
@Alkarex Is this available in the specific RSS feed view? In my concept I suggested that (after I add the RSS feed):
I can configure the view in specific newly added RSS feed eg. After I download items and read everything I can stop showing selected items that I'm not interested in (but I don't want to remember or star showed items).
@pL-uTo I guess You didn't understand my concept either 😅

**Reply by pL-uTo** [reply]
> Do what?

You're right, I should be more familiar with how to display articles in your application. Sorry

**Reply by Alkarex** [reply]
@ZicPL There is no such thing as download or not download, except for fetching full article content. If you do not want to see an article anymore, you mark it as read. If you want to easily find it again later, put it in favourites or give it a label.

Try this option:
![image](https://github.com/user-attachments/assets/a7b7f249-7ea9-4ec6-94dc-f8b1bd86b5a4)

**Reply by ZicPL** [reply]
> @ZicPL There is no such thing as download or not download, except for fetching full article content. If you do not want to see an article anymore, you mark it as read. If you want to easily find it again later, put it in favourites or give it a label.

Yes I agree with You. I saw this section in settings tab. I can say that I proposed a concept for "Adjust Showing". @Alkarex You wrote that it isn't about downloading items. It can be done as before. I proposed in concept to adjust the view for a specific RSS channel so that there are more possibilities in displaying what has been downloaded (without deleting what has been downloaded). The rest of the options "Show all articles" and "show only unread" are a specific use case. In the concept, the proposed use case is deciding to display single selected items in the view of a specific channel (without labels).

**Reply by Jeroslav** [reply]
> Yes I agree with You. I saw this section in settings tab. I can say that I proposed a concept for "Adjust Showing". @Alkarex You wrote that it isn't about downloading items. It can be done as before. I proposed in concept to adjust the view for a specific RSS channel so that there are more possibilities in displaying what has been downloaded (without deleting what has been downloaded). The rest of the options "Show all articles" and "show only unread" are a specific use case. In the concept, the proposed use case is deciding to display single selected items in the view of a specific channel (without labels).

I got notifications 🔔 
@ZicPL so you mean to select "show all" to be able to see everything that has downloaded and you don't care about: highlighting, adding a label, marking with a star?
You just want to be able to not show selected articles (in a specific RSS channel) in this specific view?

**Reply by ZicPL** [reply]
> I got notifications 🔔 @ZicPL so you mean to select "show all" to be able to see everything that has downloaded and you don't care about: highlighting, adding a label, marking with a star? You just want to be able to not show selected articles (in a specific RSS channel) in this specific view?

That's right. I suggested this in my concept. It's not easy to describe so please be understanding.
When a user is in a tab of a **specific** RSS channel of the FreshRSS app, while displaying **all** downloaded articles, He/She can hide **selected** items (articles) without the need to mark them with a label, star, etc.

**Reply by ZicPL** [reply]
@Alkarex You correctly pointed out @pL-uTo that She/He should get to know FreshRSS app better (You suggested that She/He use available options).
@Alkarex I thoroughly checked and tested FreshRSS app. I proposed a concept different from what @pL-uTo wrote about. My concept was created because it can't be set using available FreshRSS options.

@pL-uTo Can You edit subject with description because I'm describing a real problem and You asked about possibilities. Thank You for Your understanding.

**Reply by pL-uTo** [reply]
@ZicPL Is it ok now?

**Comment by ZicPL** [comment]
### New concept :

[Description](https://github.com/FreshRSS/FreshRSS/discussions/7078#discussioncomment-11522826)

![new_concept](https://github.com/user-attachments/assets/7c98aa30-ef9e-45bf-bb82-fc2199461dec)

**Reply by pL-uTo** [reply]
@ZicPL this discussion has been closed by @Alkarex  Create your own discussion :)

**Reply by ZicPL** [reply]
😠 ..

**Reply by Alkarex** [reply]
@ZicPL Feel free to open a discussion and be as specific as possible. Also, it is important to **first** explain the need / use-case, before suggesting an implementation.
You do not seem to try or answer my suggestions, which are based on what exists already.
In particular, it is not clear to me why you cannot just use the existing states we have: *read / unread / favourite (starred) / labelled (tagged)*.
The *hide* menu you proposes can already be done by clicking on read / unread and using the appropriate view mode:
![image](https://github.com/user-attachments/assets/a7b7f249-7ea9-4ec6-94dc-f8b1bd86b5a4)

**Reply by ZicPL** [reply]
> .. Feel free to open a discussion and be as specific as possible. (..)

@Alkarex  I read every comment You made and none of them relate to what I want to achieve (other people were also keeping Your attention). I try my best to describe meaning of my idea, but it isn't easy and I apologize for doing it clumsily 😢 
I tried all the settings You suggested, but none of them allow me to hide **selected** article. Everything is related to the group, not the **individual** (single) article. Seriously, if FreshRSS offered such a feature, I wouldn't waste Your time.

To preempt the question:
1. What I wrote above: it's about showing/hiding a specific article or several, but **selected** by user.
2. I consider labels to be a higher level of collection management, so in this particular case I mean the level where this collection is created (added RSS feed in the FreshRSS app).

This graphic shows where you could view hidden articles (items).
![hide_list](https://github.com/user-attachments/assets/a4c1b09a-8d6a-41d5-adce-c65978148c02)

**Reply by Mich3lleV** [reply]
As Alkarex wrote, I also think it's an unusual solution, but I see that it makes sense too. Combining Zic request with labels could give something good. Maybe it's worth opening this discussion for list pros and cons of such an approach?

**Reply by 5murfete** [reply]
Definitely it doesn't require changes in tables of database. that's how I see it as frontend dev. I don't know FreshRSS code, but anyone should know that implementation also costs time. Each item (object) has some handle so its operation should be adapted to the rest of code logic.

**Reply by Alkarex** [reply]
> none of them allow me to hide **selected** article.

> 1. it's about showing/hiding a specific article or several, but **selected** by user.

This is exactly what the read / unread state does

**Reply by ZicPL** [reply]
> Definitely it doesn't require changes in tables of database.(..)

I absolutely don't expect implementation, although I think that I'm not the only one who would be very happy if it was implemented. I wanted to consult my idea with @Alkarex because he is a freshRSS expert (Guru) and I value his opinion very much, but so far I haven't been able to clearly describe what the goal is. I really would like to be understood. I cannot explain the idea any better.

**Reply by Alkarex** [reply]
@ZicPL It is quite likely that I have not fully understood your need. But it is also possible that you are not using the current features to their full potential to address your needs.

Maybe you could try to answer the question: why is it not sufficient with the current articles states of: *read / unread / favourite (starred) / labelled (tagged)*, especially when combined with the following option:

![image](https://github.com/user-attachments/assets/a7b7f249-7ea9-4ec6-94dc-f8b1bd86b5a4)

**Reply by ZicPL** [reply]
> > none of them allow me to hide **selected** article.
> 
> > 1. it's about showing/hiding a specific article or several, but **selected** by user.
> 
> This is exactly what the read / unread state does

@Alkarex Are You sure? I tested all configurations and no effect. Ok let me show You all the cases.


Two different examples for an RSS feed from FreshRSS:
1. This example contains one unread message. "FreshRSS 1.24.0" is read.
![before_read_all](https://github.com/user-attachments/assets/bb6f8010-aed6-43e5-b427-2f893260be91)

### Goal: In FreshRSS app I want to see all new unread articles from RSS feed + one specific one e.g. "FreshRSS 1.24.0". "FreshRSS 1.24.0" is read and I don't want it to be unread.

- Settings / reading / show all articles:
![before_read_all_and_show_all_articles](https://github.com/user-attachments/assets/116d44af-8d43-40c0-b70a-f2a78b87a530)
- Settings / reading / show unread articles (with/without checked "Hide categories & feeds with no unread articles (does not work with “Show all articles” configuration)" give the same result):
![before_read_all_show_unread_arts_with_without](https://github.com/user-attachments/assets/3ee1146b-f520-4934-b8d8-efc30f797267)

Summary:
**Unable to display unread message with only one read message: FreshRSS 1.24.0". Failure.**

2. This example shows all articles read (also "FreshRSS 1.24.0".)
![after_read_all](https://github.com/user-attachments/assets/d15d31e4-f847-4fc3-862e-2eed229ee6e4)

- Settings / reading / show all articles:
![after_all_show_all_articles](https://github.com/user-attachments/assets/eb50aabd-7530-4f76-97e0-eaae7b643867)

- Settings / reading / show unread articles (with checked "Hide categories & feeds with no unread articles (does not work with “Show all articles” configuration)" give the same result):
![after_all_show_unread_articles_with_Hide_cat](https://github.com/user-attachments/assets/8386538a-c2f6-4ec4-99e3-836d32730405)

RSS feed stopped displaying completely.


- Settings / reading / show unread articles (without checked "Hide categories & feeds with no unread articles (does not work with “Show all articles” configuration)" give the same result):
![after_all_show_unread_articles](https://github.com/user-attachments/assets/193856c7-c409-4530-8771-b6284b2dda2a)

RSS feed is displayed in list but without articles..

Summary:
Cannot display just one "FreshRSS 1.24.0" (our target) unread article by itself. Failure.

I tested all states and not just "unread state does. You can clearly see that I cannot display the Read article ""FreshRSS 1.24.0"" with:
1. List of unread articles
2. without list of read articles

### I've proven that it doesn't work as You claim. Could You prove that it works for You?

__________________________________________________________________________________________________________________________________________

> (..): why is it not sufficient with the current articles states of: read / unread / favourite (starred) / labelled (tagged), especially when combined with the following option:
Yes, although I thought I had written earlier why hiding the selected article directly in the tab of a specific RSS channel is the best.

"articles states of read / unread" - inability to display the read article independently.

"articles states of favourite (starred)" - Articles marked by me with a star are important: to remember and not to delete. In contrast, selected articles (those that I do not hide) directly displayed on the RSS feed are not important to remember "forever" and can be deleted, but I have a plan for them (group work, I intend to come back to this article in the near future) - I cannot mark them as unread because with the large scale of messages that I read it is very inconvenient and causes clutter. On the other hand, articles hidden by me are completely useless and should be invisible even if they have just been downloaded.

"articles states of labelled (tagged)" - adding articles from RSS feeds is done using: Filtering articles such as date, expression, etc. This is a great tool for automation, but it isn't possible to create filters for everything (it makes no sense). This is in conflict with individual selection based on variable requirements (or individual user evaluation). My idea gives users a substitute for manual articles management.

I understand that many users don't share my optimism and I don't require it, but I gave rational reasons and described in detail the concept, which as You can see is of interest to FreshRSS users. Thank You Mr Alexandre for listening to me and Thank You for Your understanding.

**Reply by Frenzie** [reply]
Alkarex is saying that you can do many things with labels and user queries.

For example if you make a label named `delete` you can always filter for `!label:delete` and ignore the standard situation.

I think that's a touch too awkward for that specific common purpose but if you combine it with some custom JS that does it automatically when you press delete you have a reasonable workaround.

Granted, Alkarex is implicitly or explicitly talking about marking those you want to "`keep`" which doesn't do it for me (because I don't normally want to see unread but I never want to see deleted), but the principle is otherwise identical.

Perhaps I should give this some more thought, because it means thanks to various functionalities added since ca. 2018 the whole endeavor has actually become rather trivial.

**Reply by Alkarex** [reply]
All right @ZicPL , that is a better explanation. There is one feature, which we cannot yet do, but which would be quite light-weight to add, namely the ability to show articles with the state *favourites OR unread*. At the moment, we can only show *favourites*, *unread*, *favourites AND unread*. I believe it would address your need without having to change much.

**Reply by Alkarex** [reply]
Pull request draft coming (see new state representing *favourite OR unread*):

![image](https://github.com/user-attachments/assets/5054914a-1996-4072-987d-d0576d0adc49)

**Reply by Alkarex** [reply, marked as answer]
Check https://github.com/FreshRSS/FreshRSS/pull/7088
For now, you need to manually add `&state=96` to the URL

**Reply by Alkarex** [reply]
(Deleted wrong comment)

**Reply by 5murfete** [reply]
I decided to get involved in the discussion

> [Frenzie](https://github.com/Frenzie): Granted, Alkarex is implicitly or explicitly talking about marking those you want to "`keep`" which doesn't do it for me (because I don't normally want to see unread but I never want to see deleted), but the principle is otherwise identical.

@Frenzie good point 👍 

I read the whole story and it is long.. 🫨 
I'm trying to understand @Alkarex reasons. I see that Mr Alexandre avoids implementing a complete deletion of downloaded items (also selected) of the RSS feed or creating custom tags, e.g. "keep" similar to "favorite" (star), but working differently.  We may not know the reasons, but we can guess:
- high implementation costs,
- code complexity,
- maximum use of existing tools.
- etc.
I think that's why @ZicPL don't push him to code it when he wrote: "_I absolutely don't expect implementation_".

Moreover, I'm grateful to You, @ZicPL, @Alkarex and others for Your active participation in this interesting discussion 🫶 
This feature also interested me very much and not long ago I wrote about it privately with Mr Zic.

**Reply by 5murfete** [reply]
> [Alkarex](https://github.com/Alkarex): Pull request draft coming (see new state representing favourite OR unread):
@Alkarex I have tested it and confirm that it works. 

I have many favorites and it's a small problem for me, but I'll get used to it bc final profit will be much, much bigger. I am just browsing your goals and "_Add a UI button to filter on this state_" looks great.

@Alex I would like to suggest adding one more point in [[New state: favorite or unread]](https://github.com/FreshRSS/FreshRSS/pull/7088), because clicking the filter button every time for me or someone who wants to have it always on (by default) is annoying.
**Can you add a checkbox in the "Configuration View (reading, Left navigation: Categories section)" settings?**
This chexbox, when checked, would work like a clicked "_button to filter_" (by default, i.e. always), regardless of the RSS feed I'm in.

![chebox_esempio](https://github.com/user-attachments/assets/3c76d79f-2126-4237-9ce5-34ce4eecde5f)

**Reply by pL-uTo** [reply]
I just woke up and this debate is still hot 😮 
I tried to do as advised above using Labels, but it doesn't work for me at all.. I see that add **&state=36** in URL, but it doesn't work.... How can I run this? I can wait until this feature appears in next versions of FreshRSS as well 😃

**Reply by 5murfete** [reply]
In my location is night, I'm so 😪
@pL-uTo This is a prototype. Wait for the new version

**Reply by Alkarex** [reply]
Tests welcome https://github.com/FreshRSS/FreshRSS/pull/7088

**Reply by ZicPL** [reply]
@5murfete  It's nice to see You in this thread and Thanks for Your activity here.
@Frenzie i I agree with You.
@Alkarex  Yesterday I was exhausted and fell asleep.. Today I had a lot to do at work... Now I can test it and be more active.

I copied lines of code (step by step) to existing files in my instance and I confirm that it works for me 😍 
Yes, this solution solves my problem that I described here (it actually uses existing categories -> "Favourites").

@Alkarex  Have You thought about replacing "Favourites" with a new type of category (flag), e.g. "keep"?
"keep" It doesn't have to be in menu like "favorites" and a read article which isn't a favorite, but we still want to see it, doesn't appear in "favorites". don't hide ("keep") =/= favorite ("Favourites")

Is it possible to create a "keep" similar to "favorite" just for the needs of the RSS feed?

**Reply by ZicPL** [reply]
### I'm writing this info for all interested in this discussion.

After consulting @Alkarex, "keep" isn't different from "unread" and "favorites" states. If there is idea for "keep" feature no to duplicate action, it can be reconsidered.
@5murfete You wrote (last month) to me that You use a script loaded in browser (You remove the HTML element / node, when it changes state to read). Look -> @Alkarex has now implemented this is what You needed and does it automatically -> [comment](https://github.com/FreshRSS/FreshRSS/pull/7088#issuecomment-2537319733).

Everyone will probably be happy with this, but there should also be a button in UI eg. Normal view (instead of going to settings) ❗ It's fast and a lot of convenience ❗

Respect and thanks to @Alkarex and @Frenzie for listening to their users. FreshRSS is getting better every day ❤️

**Reply by 5murfete** [reply]
@ZicPL finally ! now it will be easier for me to find myself on a small screen ❤️

**Comment by ArataKuge** [comment]
Will this option be available in the next version of FreshRSS?

**Reply by 5murfete** [reply]
> Will this option be available in the next version of FreshRSS?

Probably the next version since it does not require much effort. @Alkarex  can you confirm?
Look here: [source](https://github.com/FreshRSS/FreshRSS/pull/7088)

**Reply by Alkarex** [reply]
> Will this option be available in the next version of FreshRSS?

Maybe, depending on the feedback and maturity. Tests welcome https://github.com/FreshRSS/FreshRSS/pull/7088

**Reply by RobyB89** [reply]
I've been waiting for this feature for a long time. I can't wait to use it ;]

**Reply by Alkarex** [reply]
@RobyB89 Now is a good time to test it and provide feedback

**Reply by RobyB89** [reply]
> @RobyB89 Now is a good time to test it and provide feedback

I know how to use Github as a client, I don't know how I would test this.

**Reply by Alkarex** [reply]
Testing a PR depends on how you installed FreshRSS in the first place, or on how you want to install it (if you make a test instance)

**Reply by RobyB89** [reply]
I installed FreshRSS according to the instructions on the project website:

> Get FreshRSS with git or [by downloading the archive](https://github.com/FreshRSS/FreshRSS/archive/latest.zip)
> Put the application somewhere on your server (expose only the ./p/ folder to the Web)
> Add write access to the ./data/ folder for the webserver user
> Access FreshRSS with your browser and follow the installation process
> ..

This is my only freshRSS. Do I need to install the trial version?
